src/Deserialization/Deserialization.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
```

Comment:
Has to be.

Reply:
😱

Reply:
We could technically minimize the code that is allowed to be unsafe by moving `UnsafeContext.cs` to its own tiny assembly with nothing else in it, since this is just a thin wrapper on `LLVMSharp.Interop` to add a few APIs for which it only provides pointer ways of interacting instead of reference ways. But it seems largely pointless to do so.